dts/bindings/mmc/st,stm32-sdmmc.yaml

```diff
@@ -39,6 +39,13 @@ properties:
       - 4
       - 8
 
+  clk-div:
+    type: int
+    default: 0
+    description: |
+        Clock division factor for SDMMC. Typically the clock operates at 25MHz so
+        a division factor of 2 would be 25MHz / 2 = 12.5MHz.
+
```

Comment:
Could be more detailed : 
The property could also include a range  (CLKDIV is a 10-bit value)
Div factor of 2 is dividing the input clock by 2, what would mean a Div factor of 0 ?
According to the ref Man 0438 of the stm32L5, 0 --> div by 1, etc